Mono.Cecil.Cil/Symbols.cs

```diff
@@ -802,6 +809,45 @@ public ISymbolReader GetSymbolReader (ModuleDefinition module, string fileName)
 			return null;
 		}
 
+		public static SymbolReaderKind GetSymbolReaderKind(string fileName)
+		{
+			try
+			{
```

Comment:
The idea with the Try pattern is that it communicates that the method is never going to throw, even if you pass junk input.

I don't have a strong opinion on it.  I'm ok with leaving it as is.